packages/salesforcedx-vscode-lwc/src/telemetry/telemetry.ts

```diff
@@ -5,9 +5,9 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { TelemetryReporter } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
 import * as util from 'util';
 import * as vscode from 'vscode';
-import TelemetryReporter from 'vscode-extension-telemetry';
```

Comment:
Note: we are using core extension TelemetryReporter in LWC extension (not `vscode-extension-telemetry`).